src/app/embed/embed.component.ts

```diff
@@ -64,4 +70,20 @@ export class EmbedComponent implements OnInit {
     this.artworkUrl = this.artworkUrl || this.feedArtworkUrl;
   }
 
+  @HostListener("window:resize", [])
+  setEmbedHeight() {
+    if (window.parent && window.parent.postMessage) {
+      window.parent.postMessage(JSON.stringify({
+        src: window.location.toString(),
+        context: 'iframe.resize',
+        height: 185
```

Comment:
We may eventually want some other choices for height.  But yeah... i see it's hardcoded all over the place at the moment.